### PR TITLE
Feature/warning utils and readme

### PR DESCRIPTION
**Pull request by oyvindhermansen**
- Updated Readme to match the new API
- Added a util function for warnings
- Added a util function for string-checks

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,24 +1,20 @@
...
-    if (typeof URI !== 'string') {
-      throw new Error(`URI has to be a string`);
-    }
+    warning(!URI, 'No URI specified.');
+    warning(!isString(URI), 'URI has to be a string');
+
     const URL = this.rootURL ? `${this.rootURL}/${URI}` : URI;
+
     if (!isValidURL(URL)) {
       if (this.rootURL) {
```
Comment (okaune) [inline src/index.js]: Use warning method here too?

File: README.md [hunk 2]
```diff
@@ -1,47 +1,47 @@
-# Fetch with Root - A fetch utility module
+## Fetcher - A fetch utility module
 
-Fetch from one rootURL with confident!
+<small>Fetch from one rootURL with confident!</small>
```
Comment (oyvindhermansen) [inline README.md]: Yep... Rookie mistake

File: src/index.js (hunk 1 above)
Comment (oyvindhermansen) [inline src/index.js]: Will implement!

File: src/index.js [hunk 3]
```diff
@@ -1,28 +1,31 @@
...
 
     if (!isValidURL(URL)) {
-      if (this.rootURL) {
-        throw new Error(`rootURL and URI produces a non-valid URL: "${URL}"`);
-      } else {
-        throw new Error(
-          `"${URL}" is not a valid URL. Initialize rootURL or specify a valid URL`
-        );
-      }
+      this.rootURL
```
Comment (okaune) [inline src/index.js]: This can't be correct because in the else state `this.rootURL`will never be `true`.

Comment (oyvindhermansen) [inline src/index.js]: Hmm.... Not quite sure how to solve this in a good way! Suggestions?

Comment (okaune) [inline src/index.js]: This should work:
```js
warning(!(this.rootURL && !isValidURL(URL)), `rootURL and URI produces a non-valid URL: "${URL}"`);
warning(!(!this.rootURL && !isValidURL(URL)), `"${URL}" is not a valid URL. Initialize rootURL or specify a valid URL`);
```

**Pull request merged**